pam/internal/adapter/nativemodel.go

```diff
@@ -817,14 +832,21 @@ func (m nativeModel) handleNewPassword() tea.Cmd {
 }
 
 func (m nativeModel) newPasswordChallenge(previousChallenge *string) tea.Cmd {
-	challengeLabel := fmt.Sprintf("%[1]s (or enter '%[2]s' to %[3]s)",
-		m.uiLayout.GetLabel(), nativeCancelKey, m.goBackActionLabel())
-	if previousChallenge != nil {
-		challengeLabel = fmt.Sprintf("Confirm password (or enter '%[1]s' to %[2]s)",
+	if previousChallenge == nil {
+		instructions := fmt.Sprintf("Enter '%[1]s' to cancel the request and %[2]s",
 			nativeCancelKey, m.goBackActionLabel())
+		title := m.selectedAuthModeLabel("Password Update")
+		if cmd := maybeSendPamError(m.sendInfo("== %s ==\n%s", title, instructions)); cmd != nil {
+			return cmd
+		}
 	}
```

Comment:
> As per commit 79f21be we've a new layout on the native model but this is
    not applied to the new password view, so follow the same rules here.

Can this "new password view" be entered with the actual broker implementation? Or is something that's only supported by the example broker?

Reply:
That's the same view we get with the oidc broker too, when a new password has to be created just after the device-authentication phase.

Reply:
Ah, right. I was confused because I didn't see the "Password Update" string anywhere, but after looking closer at the code, I understand that this is the only the fallback, and the msentraid broker returns "Define your local password" which is used instead.